### PR TITLE
feat: Implement _mm_cmpunord_[pd|sd]

### PR DESCRIPTION
**Pull request by HowJMay**

### CODE REVIEW

File: sse2neon.h [hunk 1]
```diff
@@ -3463,6 +3463,58 @@ FORCE_INLINE __m128d _mm_cmpord_sd(__m128d a, __m128d b)
 #endif
 }
 
+FORCE_INLINE __m128d _mm_cmpunord_pd(__m128d a, __m128d b)
```
Comment (jserv) [inline sse2neon.h:3466]: Add the descriptions from Intel Intrinsics website.

File: sse2neon.h [hunk 2]
```diff
@@ -3463,6 +3463,61 @@ FORCE_INLINE __m128d _mm_cmpord_sd(__m128d a, __m128d b)
 #endif
 }
 
+// Compare packed double-precision (64-bit) floating-point elements in a and b
+// to see if either is NaN, and store the results in dst.
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_cmpunord_pd
+FORCE_INLINE __m128d _mm_cmpunord_pd(__m128d a, __m128d b)
+{
+#if defined(__aarch64__)
+    // Excluding NaNs, any two floating point numbers can be compared.
```
Comment (marktwtn) [inline sse2neon.h:3472]: NaNs can be compared.
Though the compared result is opposite to ordinary comparison.
You should clarify the behaviour of comparing with NaN and comparing with non-NaN.

**Review by marktwtn — CHANGES_REQUESTED** [review]

File: sse2neon.h (hunk 2 above)
Comment (HowJMay) [reply sse2neon.h:3472]: How about `Two NaNs are not equal with comparison operation`?

Comment (marktwtn) [reply sse2neon.h:3472]: That will do.
Please change `with comparison operation` with `in comparison operation`.

**Review by marktwtn — APPROVED** [review]

**Pull request merged**